### PR TITLE
Add typescript typedef file

### PR DESCRIPTION
**Pull request by MichaelRBond**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (tkporter) [inline index.d.ts]: nit: can you remove this newline? happy to merge & publish after that 👍

Comment (MichaelRBond) [reply index.d.ts]: Removed. Thanks.

**Comment by tkporter** [comment]
Thanks for the PR! Just published with the changes under version 1.6.0

**Comment by MichaelRBond** [comment]
thanks

**Pull request merged**